docs/self-managed/operational-guides/troubleshooting/troubleshooting.md

```diff
@@ -66,3 +66,87 @@ A gateway timeout can occur if the headers of a response are too big (for exampl
 ## Helm CLI version and installation failures
 
 If you encounter errors during Helm chart installation, such as type mismatches or other template rendering issues, you may be using an outdated version of the Helm CLI. Helm's handling of data types and template syntax can vary significantly between versions. Ensure you use the Helm CLI version `3.13` or higher.
+
+## Anomaly detection scripts
+
+The [c8-sm-checks](https://github.com/camunda/c8-sm-checks) project introduces a set of scripts to aid detection of Camunda deployment anomalies.
+
+These scripts perform health checks on various aspects of the Kubernetes installation and Zeebe components, providing insights into potential issues that may affect the performance or stability.
+
+### Usage
+
+Each script in the `c8-sm-checks` project can be executed independently, allowing you to target specific areas for troubleshooting and verification.
+
+To utilize these scripts effectively, ensure you have the necessary permissions and access to your Kubernetes cluster. Additionally, make sure you have the required dependencies installed on your system, such as `kubectl`, `helm`, `curl`, and `grpcurl`.
```

Comment:
I'd probably rephrase this part about `necessary permissions` since we don't mention anything what the minimal set of permissions is so just saying make sure to have something of which no one knows what it is, is not gonna help.

Probably rephrase it to just the following or something similar.

```
.., ensure you have access to your Kubernetes cluster.
```

Reply:
makes sense 👍 , thanks for the adjustments!